common/utils.cpp

```diff
@@ -34,6 +32,8 @@
 #include <cstdlib>
 #include <limits>
 
+//put it after va.h to fix the I420 redefine issue
+#include "utils.h"
```

Comment:
This change is not necessary since you've added the va/va.h include in the utils.h header.  You should also remove the va/va.h include in utils.cpp since it is redundant now.

Reply:
okay, Updated, thanks @uartie .